pom.xml

```diff
@@ -198,6 +200,78 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.rat</groupId>
+        <artifactId>apache-rat-plugin</artifactId>
+        <version>0.12</version>
```

Comment:
Do not specify the version explicitly, the one from the parent Apache pom should be used.